Sprint-2/debug/1.js

```diff
@@ -1,8 +1,19 @@
 // Predict and explain first...
+// Prediction : it will return "the sum of 10 and 32 is null". Because the function returns something but it's null.
 
 function sum(a, b) {
   return;
   a + b;
 }
 
 console.log(`The sum of 10 and 32 is ${sum(10, 32)}`);
+// The result of summing 10 and 32 is 320
+
+// Predict and explain first...
+
+function sum(a, b) {
+  return;
+  a + b;
+}
+
+console.log(`The sum of 10 and 32 is ${sum(10, 32)}`);
```

Comment:
#### Todo
The current script still does not show the sum as 42. You also need to correct the code in the function.